architecture/activity-login.puml

```diff
@@ -0,0 +1,20 @@
+@startuml Login activity diagram
+start
+if (Is logged in?) then (yes)
+else (no)
+  :Login form;
+  if (Is registered?) then (yes)
```

Comment:
We wouldn't ask this question to our service, as it's a security issue to reveal whether an email is registered.